kubernetes/istio.go

```diff
@@ -30,7 +30,9 @@ const (
 
 var (
 	portNameMatcher = regexp.MustCompile(`^[\-].*`)
-	portProtocols   = [...]string{"grpc", "http", "http2", "https", "mongo", "redis", "tcp", "tls", "udp", "mysql"}
+	portProtocols   = map[string][]int32{"grpc": {443}, "http": {80}, "http2": {443, 80}, "https": {443},
+		"mongo": {27017, 27018, 27019}, "redis": {6379}, "tcp": {}, "tls": {}, "udp": {},
```

Comment:
I'm reviewing this PR. @hhovsepy do you remember how Istio defines these mappings.
I remember this was something defined at the Envoy level, but those names/numbers were defined in some variables?